src/tooltip/definition-tooptip.component.ts

```diff
@@ -57,6 +57,10 @@ export class TooltipDefinition extends PopoverContainer {
 	 * The string or template content to be exposed by the tooltip.
 	 */
 	@Input() description: string | TemplateRef<any>;
+	/**
+	 * Optional data for templates
+	 */
+	@Input() data = {}
```

Comment:
Let's use `context` or perhaps `templateContext` to make it more descriptive.

```suggestion
	/**
	 * Optional data for templates
	 */
	@Input() context: any;
```